README.md

```diff
@@ -94,27 +94,57 @@ MEDS data also must satisfy two important properties:
     in one and only one parquet file.
 2. Data about a single subject must be contiguous within a particular parquet file and sorted by time.
 
-The data schema has four mandatory fields:
+The data schema has four fields that will appear in every MEDS dataset data file:
 
 1. `subject_id`: The ID of the subject this event is about.
 2. `time`: The time of the event. This field is nullable for static events.
 3. `code`: The code of the event.
 4. `numeric_value`: The numeric value of the event. This field is nullable for non-numeric events.
 
-In addition, it can contain any number of custom properties to further enrich observations. The python
-function below generates a pyarrow schema for a given set of custom properties.
+In addition, it can contain any number of custom properties to further enrich observations. This is reflected
+programmatically using the [`flexible_schema`](<>) package as the below schema:
```

Comment:
```suggestion
programmatically using the [`flexible_schema`](https://github.com/mmcdermott/flexible_schema) package as the below schema:
```

Reply:
I would maybe also mention that this is a mixin/decorator hence the unusual syntax